infrastructure/main.tf

```
@@ -52,6 +51,39 @@ module "ccpay-bulkscanning-payment-database" {
   common_tags = "${var.common_tags}"
 }
 
+# Populate Vault with DB info
+
+resource "azurerm_key_vault_secret" "POSTGRES-USER" {
+  name      = "${var.component}-POSTGRES-USER"
+  value     = "${module.ccpay-bulkscanning-payment-database.user_name}"
+  vault_uri = "${data.azurerm_key_vault.payment_key_vault.id}"
```

don't forget the left hand side too
```suggestion
  key_vault_id = "${data.azurerm_key_vault.payment_key_vault.id}"
```

---

Please check now it should be good